Bio/SubsMat/MatrixInfo.py

```diff
@@ -35,2888 +37,2691 @@
 
```

Comment:
The comment above can now be removed.

Reply:
Good catch, yes - removing the ``_temp`` functions was the main change in the pull request.